CMakeLists.txt

```diff
@@ -711,7 +711,9 @@ target_include_directories(${MINIZIP_TARGET} PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>)
 if(MZ_COMPAT)
     target_include_directories(${MINIZIP_TARGET} PUBLIC
-        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>)
+        $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}>
+        ${ZLIB_SOURCE_DIR}
+        ${ZLIB_BINARY_DIR})
```

Comment:
From my experiments, that part isn't needed. It's actually causing extra build issues (for Windows).

I came up with a solution that builds in #815